newIDE/app/src/UI/Theme/DefaultDarkTheme/theme.json

```diff
@@ -128,16 +128,6 @@
         }
       },
       "canvas": {
-        "events": {
```

Comment:
removed on purpose?

Reply:
Yes, I don't why those 2 themes had those lines, it's not used in the CreateTheme file